CHANGELOG.md

```diff
@@ -12,4 +12,24 @@ Bugfixes:
 
 Other improvements:
 
-## [0.0.0] - 2020-01-01
+## [v4.0.1](https://github.com/purescript-contrib/purescript-js-timers/releases/tag/v4.0.1) - 2018-06-23
+
+- Adds metadata including contributor guidelines
+- Pushes latest release to Pursuit
+
+## [v4.0.0](https://github.com/purescript-contrib/purescript-js-timers/releases/tag/v4.0.0) - 2018-05-26
+
+Updates for 0.12
+
+## [v3.0.0](https://github.com/purescript-contrib/purescript-js-timers/releases/tag/v3.0.0) - 2017-04-03
+
+- Updated for PureScript 0.11
+
+## [v2.0.0](https://github.com/purescript-contrib/purescript-js-timers/releases/tag/v2.0.0) - 2016-10-17
+
+- Updated dependencies
+
+## [v1.0.0](https://github.com/purescript-contrib/purescript-js-timers/releases/tag/v1.0.0) - 2016-07-25
+
+Initial release
+
```

Comment:
```suggestion
```

Looks like these are ending up with an extra trailing newline -- I'm removing them, but FYI for the script itself.